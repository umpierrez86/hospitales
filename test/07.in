leer_instancia
7 12
1 1 3 2 1 1 1
0 1 2 3 4 5 6 7 8 9 10 11
0 1 2 3 4 5 6 7 8 9 10 11
0 1 2 3 4 5 6 7 8 9 10 11
11 10 9 8 7 6 5 4 3 2 1 0
11 10 9 8 7 6 5 4 3 2 1 0
11 10 9 8 7 6 5 4 3 2 1 0
5 4 3 2 1 0 11 10 9 8 7 6
0 1 2 3 4 5 6
3 1 0 2 5 6 4
1 3 2 0 4 5 6
1 0 2 3 5 6 4
4 5 6 2 0 1 3
4 5 6 0 1 2 3
5 6 4 3 1 0 2
5 6 4 1 3 2 0
4 5 6 1 0 2 3 
2 0 1 3 5 6 4
2 0 1 3 5 6 4
2 0 1 3 5 6 4
imprimir_instancia
asignar_residencias
imprimir_asignacion
Fin