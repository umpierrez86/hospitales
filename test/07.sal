valgrind: No such file or directory
