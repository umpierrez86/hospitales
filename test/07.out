1>2>Cantidad hospitales: 7
Cupos hospitales: 1 1 3 2 1 1 1 
Cantidad estudiantes: 12
Preferencias hospitales
Preferencias hospital h0: 0 1 2 3 4 5 6 7 8 9 10 11 
Preferencias hospital h1: 0 1 2 3 4 5 6 7 8 9 10 11 
Preferencias hospital h2: 0 1 2 3 4 5 6 7 8 9 10 11 
Preferencias hospital h3: 11 10 9 8 7 6 5 4 3 2 1 0 
Preferencias hospital h4: 11 10 9 8 7 6 5 4 3 2 1 0 
Preferencias hospital h5: 11 10 9 8 7 6 5 4 3 2 1 0 
Preferencias hospital h6: 5 4 3 2 1 0 11 10 9 8 7 6 
Preferencias estudiantes
Preferencias estudiante e0: 0 1 2 3 4 5 6 
Preferencias estudiante e1: 3 1 0 2 5 6 4 
Preferencias estudiante e2: 1 3 2 0 4 5 6 
Preferencias estudiante e3: 1 0 2 3 5 6 4 
Preferencias estudiante e4: 4 5 6 2 0 1 3 
Preferencias estudiante e5: 4 5 6 0 1 2 3 
Preferencias estudiante e6: 5 6 4 3 1 0 2 
Preferencias estudiante e7: 5 6 4 1 3 2 0 
Preferencias estudiante e8: 4 5 6 1 0 2 3 
Preferencias estudiante e9: 2 0 1 3 5 6 4 
Preferencias estudiante e10: 2 0 1 3 5 6 4 
Preferencias estudiante e11: 2 0 1 3 5 6 4 
3>4>(0, 0) (1, 1) (2, 2) (3, 2) (4, 2) (5, 6) (6, libre) (7, libre) (8, 4) (9, 5) (10, 3) (11, 3) 
5>Fin.
